AcceptanceTests/AcceptanceTests.swift

```diff
@@ -99,6 +99,18 @@ class AcceptanceTests: QuickSpec {
                     }
                 }
                 
+                context("with --files-to-mutate as an argument") {
+                    var output: String!
+                    
+                    beforeEach {
+                        output = self.muterFilesToMutateOutput
+                    }
+                    
+                    they("only mutate the given list") {
+                        expect(self.numberOfDiscoveredFileLists(in: output)).to(equal(1))
```

Comment:
`self.numberOfDiscoveredFileLists(in:)` has a deceptive name - i don't think this is the method you're looking for. This just proves that Muter logs of filelist, but it doesn't prove anything about the number of files contained in that list.

This line should be updated to `expect(output.contains("In total, Muter discovered 1 mutants in 1 files")).to(beTrue())` which will prove that Muter is using the command line flag you added.